scripts/feature_tests/upgrade/upgrade.sh

```diff
@@ -42,11 +42,10 @@ pushd "${METAL3_DEV_ENV_DIR}/scripts/feature_tests/upgrade/controlplane_upgrade"
 source 3cp_1w_k8sVer_bootDiskImage_scaleInWorker_upgrade.sh
 popd || exit
 
-# This needs to be replaced by 1cp_1w_bootDiskImageANDK8sCotrollers_clusterLevel_upgrade.sh
 # Run controlplane components upgrade tests | This should be the last one
```

Comment:
change the comment to reflect the current changes.

Reply:
done